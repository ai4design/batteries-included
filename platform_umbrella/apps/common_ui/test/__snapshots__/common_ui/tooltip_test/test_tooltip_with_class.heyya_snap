<div class="group relative underline">
  <div class="invisible group-hover:visible group-focus-within:visible absolute -translate-x-1 -translate-y-full rounded-md shadow-sm bg-pink-200 bg-opacity-95 p-2" role="tooltip" aria-hidden="true">
    Test
  </div>
  
  Underlined text

</div>