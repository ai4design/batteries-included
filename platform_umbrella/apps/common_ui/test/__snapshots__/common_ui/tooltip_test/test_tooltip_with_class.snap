"<div class=\"group relative underline\">\n  <div class=\"invisible group-hover:visible group-focus-within:visible absolute -translate-x-1 -translate-y-full rounded-md shadow-sm bg-pink-200 bg-opacity-95 p-2\" role=\"tooltip\" aria-hidden=\"true\">\n    Test\n  </div>\n  \n  Underlined text\n\n</div>"